game/scripts/npc/heroes/techies.txt

```diff
@@ -5,9 +5,9 @@
   //=================================================================================================================
   "npc_dota_hero_techies"
   {
-    "Ability13"                                           "special_bonus_intelligence_35" // replaces special_bonus_mp_regen_3
-
     "AttackDamageMin"                                     "24" // 9
     "AttackDamageMax"                                     "26" // 11
-  }
+    "Ability13"                                           "special_bonus_all_stats_15" // replaces special_bonus_mp_regen_3
+    "Ability17"						  "special_bonus_cast_speed_30" // temporarily replaces mine movement. Doesn't stack with arcane blink
```

Comment:
I HATE movement on mines. It ruins the design of the hero, especially with it at such an absurd number. I gave him a 30% cast speed boost and honestly through testing it suchs a huge QOL change. Not OP but in these late-game fights and duels techies suffers at using his spells since they all have huge cast point. This doesn't push him over any edges but makes his ability to use his spells way more viable. This is the one change I'm hardstuck on and will not change!!!

Reply:
Does this talent even work?

Reply:
And also doesn't aghs also reduces cast time enough already.

Reply:
it works, and aghs doesnt reduce cast time